src/enclave/enclave.h

```diff
@@ -24,6 +24,11 @@
 
 namespace enclave
 {
+  inline void from_json(const nlohmann::json& j, SizeString& str)
+  {
+    str = j.get<size_t>();
```

Comment:
As discussed, this was the case because we didn't want to take a dependency on CLI11 in the enclave. I've re-implemented it so we no longer rely on CLI11 and can use the same JSON serder logic in both the host and the enclave.